prow/cmd/phony/examples/pr.json

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
Could you give this file a more descriptive name such as `opened_pr.json` or whatever the action is.